styles/components/navigation.css

```
@@ -0,0 +1,9 @@
+.navigation {
+  border: 1px solid saddlebrown;
+  box-shadow: inset 0 0 1rem saddlebrown;
+}
+
+.navigation__link--font {
+  color: seashell;
+  font-weight: 500;
+}
```

This is not really a modifier of your links in the `navigation`, so I would just get rid of the `--font` :)
```suggestion
.navigation__link {
  color: seashell;
  font-weight: 500;
}
```